sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/Credential/ServicePrincipalDatasourceCredential.cs

```diff
@@ -53,18 +53,6 @@ internal ServicePrincipalDatasourceCredential(DataSourceCredentialType dataSourc
         /// </summary>
         public string TenantId { get; set; }
 
-        /// <summary>
```

Comment:
Just removing the method to the end of the properties. Didn't want to mix them up.